docs/api/schemas.md

```diff
@@ -80,15 +80,37 @@
 
 ```json
 {
-  "report_interval": 0
+  "report_interval": -9223372036854776000
 }
 ```
 
 ### Properties
 
-| Name              | Type    | Required | Restrictions | Description                                                                    |
-| ----------------- | ------- | -------- | ------------ | ------------------------------------------------------------------------------ |
-| `report_interval` | integer | false    |              | Report interval is the duration after which the agent should send stats again. |
+| Name              | Type                           | Required | Restrictions | Description                                                                    |
+| ----------------- | ------------------------------ | -------- | ------------ | ------------------------------------------------------------------------------ |
+| `report_interval` | [time.Duration](#timeduration) | false    |              | Report interval is the duration after which the agent should send stats again. |
+
+## codersdk.AuditAction
+
+```json
+"create"
+```
+
+### Properties
+
+| Name          | Type   | Required | Restrictions | Description |
+| ------------- | ------ | -------- | ------------ | ----------- |
+| `*anonymous*` | string | false    |              |             |
+
+#### Enumerated Values
+
+| Property      | Value    |
+| ------------- | -------- |
+| `*anonymous*` | `create` |
+| `*anonymous*` | `write`  |
+| `*anonymous*` | `delete` |
+| `*anonymous*` | `start`  |
+| `*anonymous*` | `stop`   |
```

Comment:
The name/property columns don't really make sense here 🤔. We might want to rewrite how enums are represented.

Reply:
I refactored them a bit. Let me know your thoughts.

Reply:
It's nice! (Optionally) we could show the base type in the enum description. But it's kind of inferable via the `json` field so not sure it's necessary.